scripts/setupTypeScript.js

```diff
@@ -3,7 +3,7 @@
 /** This script modifies the project to support TS code in .svelte files like:
 
   <script lang="ts">
-  	export let name: string;
+    export let name: string;
```

Comment:
The style guide for Svelte is to use tabs, not spaces.

Reply:
I don't mind reverting this -- this is just _Format on Save_.  However, where is that _Svelte Style Guide_ documented?  (I couldn't find anything.) And would that apply to this `js` script (since it isn't a `.svelte` file)?